### ISSUE TITLE
Skip the Ecore options dialog when a valid ecore settings resource exists

### ISSUE THREAD

**Issue by rensink** [nl-utwente-groove/code#558]
Ecore imporing/exporting involves a lot of superfluous steps:
- Choosing the ecore metamodel
- Going through the configuration dialog
- OKing the ecore type graph

All these should be necessary only once.

Reported by: rensink

**Comment by rensink** [comment]
Status as of 2026-08, after the Ecore import/export was rebuilt on modern EMF (f85d56128 and follow-ups, `io/external/format/ecore`): two of the three superfluous steps are gone.

- *Choosing the ecore metamodel*: eliminated. `EcorePorter` auto-scans sibling `.ecore` files in the imported file's directory and resolves the metamodel through EMF's package registry; on export the metamodel comes from the grammar's active type graphs.
- *OKing the ecore type graph*: eliminated. Importing an `.xmi` yields the TYPE and HOST resources in one go, without a separate confirmation.
- *The configuration dialog*: still shown unconditionally on every import and export (`SimulatorAction.askEcoreOptions`, called from `ImportAction` and `ExportAction`). It does now persist its answers into the grammar's `ecore` settings resource, so it opens pre-seeded and hand-written per-element entries survive — but the modal round-trip itself remains.

Rescoping this issue to the remaining step: skip the dialog when a valid `ecore` settings resource already exists (offering it only on first import, or behind an explicit button), since the settings display now makes the values directly editable anyway.

**Comment by rensink** [comment]
Resolved by 8d6c233ce, merged into master as 9b92ebbe2: the options dialog now appears only on an Ecore import into a grammar without a mapping resource, and creates that resource; exports never ask (they use the resource if present, the defaults otherwise). The remaining Ecore settings follow-ups (type-graph compatibility check, nsURI drift warning) are tracked as items 6 and 7 of gh #898.
